dotnetv3/DynamoDB/scenarios/PartiQL_Batch_Scenario/PartiQL_Batch_Scenario/PartiQLBatchMethods.cs

```diff
@@ -0,0 +1,197 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+namespace PartiQL_Batch_Scenario
+{
+    public class PartiQLBatchMethods
+    {
+        private static readonly AmazonDynamoDBClient Client = new AmazonDynamoDBClient();
+
+        // snippet-start:[PartiQL.dotnetv3.PartiQLBasicsScenario-InsertMovies]
+
+        /// <summary>
+        /// Inserts movies imported from a JSON file into the movie table using
+        /// a DynamoDB PartiQL INSERT statement.
+        /// </summary>
+        /// <param name="tableName">The name of the table where the movie
+        /// information will be inserted.</param>
+        /// <param name="movieFileName">The name of the JSON file containing
+        /// movie information.</param>
+        /// <returns>A Boolean value indicating the success or failure of the
+        /// insert operation.</returns>
+        public static async Task<bool> InsertMovies(string tableName, string movieFileName)
+        {
+            // Get the list of movies from the JSON file.
+            var movies = ImportMovies(movieFileName);
+
+            var success = false;
+
+            if (movies is not null)
+            {
+                // Insert the movies in a batch using PartiQL. The batch can only contain
+                // a maxium of 25 items, so insert 25 movies at a time.
```

Comment:
Insert the movies in a batch by using PartiQL. Because the batch can contain a maximum of 25 items, insert 25 movies at a time.

Reply:
changed